LaTeX/DocStrip.sublime-syntax

```diff
@@ -19,35 +19,34 @@ contexts:
 
   controls:
     - meta_append: true
-    - match: ((\\)(?:endbatchfile)){{endcs}}
+    - match: (\\)(?:endbatchfile){{endcs}}
+      scope: keyword.control.tex.docstrip
       captures:
-        1: keyword.control.tex.docstrip
-        2: punctuation.definition.backslash.tex
-    - match: ((\\)(?:(?:batch)?input)){{endcs}}
-      scope: meta.function.input.tex
+        1: punctuation.definition.backslash.tex
+    - match: (\\)(?:(?:batch)?input){{endcs}}
+      scope: meta.function.input.tex keyword.control.input.tex
       captures:
-        1: keyword.control.input.tex
-        2: punctuation.definition.backslash.tex
-    - match: ((\\)ifToplevel){{endcs}}
+        1: punctuation.definition.backslash.tex
+    - match: (\\)ifToplevel{{endcs}}
+      scope: keyword.control.conditional.tex.docstrip
       captures:
-        1: keyword.control.conditional.tex.docstrip
-        2: punctuation.definition.backslash.tex
+        1: punctuation.definition.backslash.tex
 
   docstrip-preamble:
-    - match: ((\\)((?:declare)?(?:pre|post)amble)){{endcs}}
+    - match: (\\)((?:declare)?(?:pre|post)amble){{endcs}}
```

Comment:
I'd suggest to either remove capture group

```suggestion
    - match: (\\)(?:declare)?(?:pre|post)amble{{endcs}}
```

or convert capture group in normal group

```suggestion
    - match: (\\)(?:(?:declare)?(?:pre|post)amble){{endcs}}
```